python/enthic/extract_bundle.py

```diff
@@ -21,6 +21,7 @@
 from os.path import dirname, join, isdir
 from re import sub, compile
 from zipfile import ZipFile, BadZipFile
+from .ontology import CODE, ONTOLOGY
```

Comment:
It work fine but please use the package path import enthic.ontology ...

Reply:
Okay

Reply:
I will include the new variable CODE in bundle_extract directly